.ci.yaml

```diff
@@ -4622,3 +4622,22 @@ targets:
       tags: >
         ["devicelab", "hostonly", "mac"]
       task_name: flutter_tool_startup__macos
+
+  # This is used to test the new release scheduler. This will eventually be deleted.
+  - name: Linux docs_test_with_release_scheduler
+    recipe: flutter/flutter
+    scheduler: release
+    bringup: true
```

Comment:
Append the bug why this is `bringup: true`, like a new target, or the target is flaky.

Reply:
Updated the PR description